src/anemoi/models/models/hierarchical.py

```diff
@@ -0,0 +1,308 @@
+# (C) Copyright 2024 ECMWF.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+#
+
```

Comment:
Great great, and nice to have this feature added! I would just like to point out that seems like a new feature that could have used some documentation. Was that added during the PR ? If not probably we could at least create a new PR to include an update to the docs

Reply:
Comment addressed on this `[PR](https://github.com/ecmwf/anemoi-models/pull/94/commits/225315ea8e4cac6480e89cbb3bbd73c543386614 )`